src/dot.rs

```diff
@@ -111,6 +112,8 @@ pub enum Config {
     NodeNoLabel,
     /// Do not print the graph/digraph string.
     GraphContentOnly,
+    /// For aing any attributes, mostly intended for: https://graphviz.org/docs/graph/
```

Comment:
What `aing` means?

Reply:
Oops! I probably meant for it to say 'adding'. Fixed.